core/src/main/java/tc/oc/pgm/filters/FilterParser.java

```diff
@@ -453,4 +454,50 @@ public TimeFilter parseTimeFilter(Element el) throws InvalidXMLException {
   public ScoreFilter parseScoreFilter(Element el) throws InvalidXMLException {
     return new ScoreFilter(XMLUtils.parseNumericRange(new Node(el), Integer.class));
   }
+
+  @MethodParser("match-phase")
+  public Filter parseMatchPhase(Element el) throws InvalidXMLException {
+    return parseMatchPhaseFilter(el.getValue(), el);
+  }
+
+  @MethodParser("match-started")
+  public Filter parseMatchStarted(Element el) throws InvalidXMLException {
+    return parseMatchPhaseFilter("started", el);
+  }
+
+  @MethodParser("match-running")
+  public Filter parseMatchRunning(Element el) throws InvalidXMLException {
+    return parseMatchPhaseFilter("running", el);
+  }
+
+  @MethodParser("match-finished")
+  public Filter parseMatchFinished(Element el) throws InvalidXMLException {
+    return parseMatchPhaseFilter("finished", el);
+  }
+
+  private Filter parseMatchPhaseFilter(String matchState, Element el) throws InvalidXMLException {
+
+    MatchPhase matchPhase = null;
+
+    switch (matchState) {
+      case "running":
+        matchPhase = MatchPhase.RUNNING;
+        break;
+      case "finished":
+        matchPhase = MatchPhase.FINISHED;
+        break;
+      case "starting":
+        matchPhase = MatchPhase.STARTING;
+        break;
+      case "idle":
+        matchPhase = MatchPhase.IDLE;
+        break;
+      case "started":
+        return AnyFilter.of(
+            new MatchPhaseFilter(MatchPhase.RUNNING), new MatchPhaseFilter(MatchPhase.FINISHED));
```

Comment:
```suggestion
      case "running": return MatchPhaseFilter.RUNNING;
      case "finished": return MatchPhaseFilter.FINISHED;
    // ...
```
As suggested in discord you should make static final fields in MatchPhaseFilter and reuse the same instances